src/components/shared/thread/ActivityGraphFills.js

```diff
@@ -238,10 +250,9 @@ export class ActivityGraphFillComputer {
         // step and eliminating all the null values.
         const cpuDeltaBefore = ensureExists(threadCPUDelta[i]);
         const cpuDeltaAfter = ensureExists(threadCPUDelta[i + 1]);
-        const intervalDistribution =
-          i === 0 ? 1 : (samples.time[i] - samples.time[i - 1]) / interval;
+        const intervalDistribution = (sampleTime - prevSampleTime) / interval;
```

Comment:
I'm happy that we could actually simplify this code and remove the usage of ternary here (and the part which is about the last sample)

Reply:
Yeah, we could to that already, but somehow I was using the `samples.time` directly instead of that variable.